README.md

```diff
@@ -125,7 +128,7 @@ NoMethodError: private method `age' called for #<Cat:0x000001020bfce0 @name="Lil
 
 ### Why Have A Private `attr_accessor`?
 
-Previously we were using getters/setters so the outside world could access our instance variables. But now we are using them so that we can locally access them. Why is that valuable? A lot of software bugs come from the incorrect modification of state, aka variables. If we can centralize all modification into one place we can simplify our debugging efforts. That's the beauty of enforcing getters/setters. Let's say you want to see all the places an instance variable is modified. With a private `attr_accessor` you can just override it and put a `binding.pry`.
+Previously we were using getters/setters so the outside world could access our instance variables. But now we are using them so that we can locally access them. Why is that valuable? Bugs can result from the incorrect modification and assignment of variables. If we can limit outside access to variables, we can simplify our debugging efforts. That's the beauty of enforcing getters/setters. Let's say you want to see all the places an instance variable is modified. With a private `attr_accessor` you can just override it and put a `binding.pry`.
```

Comment:
I know this wasn't you and that I didn't create an issue about this, but I think we shouldn't talk about "overriding" when what this lesson is asking us to do is rewrite the attr_accessor. So maybe we should ditch the private attr_accessor? Is it just me or does that just feel kind of wrong?